package.json

```diff
@@ -74,7 +74,7 @@
 		"@wordpress/i18n": "^3.9.0",
 		"@wordpress/is-shallow-equal": "^1.8.0",
 		"accounting-js": "^1.1.1",
-		"antd": "^3.26.9",
+		"antd": "^4.0.0-rc.5",
```

Comment:
This is the latest version:
![image](https://user-images.githubusercontent.com/1477453/74866290-4ef48b00-535b-11ea-9214-f1af8d322688.png)